_observing-your-data/trace/ta-dashboards.md

```diff
@@ -10,16 +10,134 @@ redirect_from:
 
 # Trace Analytics OpenSearch Dashboards plugin
 
-The Trace Analytics plugin for OpenSearch Dashboards provides at-a-glance visibility into your application performance, along with the ability to drill down on individual traces. For installation instructions, see [Standalone OpenSearch Dashboards plugin install]({{site.url}}{{site.baseurl}}/install-and-configure/install-dashboards/plugins/).
+The Trace Analytics plugin offers at-a-glance visibility into application performance based on [OpenTelemetry (OTel)](https://opentelemetry.io/) protocol data. OTel, an open-source observability framework, offers standardized instrumentation for collecting telemetry data from cloud-native software.
 
-The **Dashboard** view groups traces together by HTTP method and path so that you can see the average latency, error rate, and trends associated with a particular operation. For a more focused view, try filtering by trace group name.
+An [OTel demo](#otel-demo) is provided at the end of this documentation.
 
```

Comment:
@DB-YANG Should the user be linked to the demo landing page instead (https://github.com/opensearch-project/opentelemetry-demo/blob/main/tutorial/DemoLandingPage.md)?